internal/health/healthdb/schema/0001_initial.sql

```diff
@@ -0,0 +1,32 @@
+-- Used to normalise identifiers for health checks.
+-- Each check id is only unique per device,
+-- so we need to combine it with the device name to get a unique key.
+CREATE TABLE health_check_ids
+(
+    id       INTEGER PRIMARY KEY AUTOINCREMENT,
```

Comment:
Would the default `INTEGER PRIMARY KEY` allocator be OK? `AUTOINCREMENT` has a (small) performance penalty. If it is needed, might be worth a comment that the id needs to be unique for all time.
See also: https://sqlite.org/autoinc.html